src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicDoNotUseCountWhenAnyCanBeUsedFixer.vb

```diff
@@ -133,18 +133,25 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
 
             End If
 
-            If invocationExpression Is Nothing Then
+            If invocationExpression IsNot Nothing Then
 
-                expression = Nothing
-                arguments = Nothing
-                Return
+                expression = DirectCast(invocationExpression.Expression, MemberAccessExpressionSyntax).Expression
+                arguments = invocationExpression.ArgumentList.ChildNodes()
+                Return True
 
             End If
 
-            expression = DirectCast(invocationExpression.Expression, MemberAccessExpressionSyntax).Expression
-            arguments = invocationExpression.ArgumentList.ChildNodes()
+            Dim memberAccessExpression = TryCast(sourceExpression, MemberAccessExpressionSyntax)
+            If memberAccessExpression IsNot Nothing Then ' This case happens for something like: x = GetData().Count where Count method is called without parentheses.
```

Comment:
Move the comment to a new line on top of line 144 so its clear that this is a special scenario.